config/overlays/basic/certificate.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: webhook-server-cert
+  namespace: system
+data:
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURiRENDQWxTZ0F3SUJBZ0lVZDQwVTlpN0YweHNZNjJPMVFnVW9tQ0tEenNVd0RRWUpLb1pJaHZjTkFRRUwKQlFBd1RqRUxNQWtHQTFVRUJoTUNWVk14RGpBTUJnTlZCQW9UQld0aFptdGhNUzh3TFFZRFZRUURFeVpyWVdacgpZUzFvWldGa2JHVnpjeTVyWVdacllTNXpkbU11WTJ4MWMzUmxjaTVzYjJOaGJEQWVGdzB4T1RBM016RXhORFUzCk1EQmFGdzB5T1RBM01qZ3hORFUzTURCYU1FNHhDekFKQmdOVkJBWVRBbFZUTVE0d0RBWURWUVFLRXdWcllXWnIKWVRFdk1DMEdBMVVFQXhNbWEyRm1hMkV0YUdWaFpHeGxjM011YTJGbWEyRXVjM1pqTG1Oc2RYTjBaWEl1Ykc5agpZV3d3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLQW9JQkFRQ3dna09FVXp5c09vUUh1c29XCmY1R1IzcEVGejZqWVl5QzZDRmZYTkxVcDNpSk5naWcrZGp1SzUyczloRUpJK08zWlArUDFtS3E2TmxBMHQyYWEKTk0zUHh3ZlVKQUkzV1VhTU5GUkdWbERWejBIZVdhS1RLZDJia2ZqUHoyTEJnZXpFYS90clBTODRBN0duODlDbApmTUw4clUzdkVmMFljWUdhNWRNeFlRbHF3elovM1pFQVJtbzRIdWUrYlRQWVlJL3BKbnZsLzJyOUppdEVVdWU5CkhhamdjNzI4WXJ6b2VaLzVyNlBNTmc5NjFKQlk3RTE5MUgrc3FpYkM3U2FBcEdMa2xabmtLaTA0UytoeTgybDgKVWpzSDNNWEc0NmtvdGc2K1IxUjBYcXo3Y1UySEpab2QwdElCaGZhdUM4VlA4bHd5REdOTjJwWndkVFptZUNKbQptUjl4QWdNQkFBR2pRakJBTUE0R0ExVWREd0VCL3dRRUF3SUJCakFQQmdOVkhSTUJBZjhFQlRBREFRSC9NQjBHCkExVWREZ1FXQkJRVW16azcveEJ2QldGOEZYNGh1eVpKbXB5dnBUQU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUEKbXcrTE9LYkRKMlBocmtDK3dIWTNnMWJnTndYaWZSWUkxYS9JaWkyTkpzOFh2blA4Y3J0dWJnZDl1bDhPQVJWaAoweEw4M3oxdzU3VHpxWm5HVXZPUXNYL3p6SWlnNFl3VUZQQ2s5RjJPRisrTnpUdFRTSFU5UFRKVit2dXMvK0R3CldJem1ocmtjOXg2a05GK29idHRWbGkyK3BqL3hOaVpBbjZHM09zWVByUG1uaVRtZUlkemowL3p1Ym9lc2pOWW8KZEtQTVZKemJKOFlQZUtKWnhkelcwQkludTlmWUpWNHpjYWR1VlFIZTkxWGR2TS9oUTdzcnIzTXIyZUNLZW4vaAp6VkpXY1ptdi94SVF5K1VLNDdxd09EcDN2YVU4NGJYeWFDMDR3RGwwUkdqeWM4M3VuSStYMG80OVgxdVRGemRvCi9sQXlZcUVkVVpLcVBiejV3RldwUXc9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
You don't need the CA in this secret I don't think, though it doesn't hurt I guess. The webhook server just needs to know it's cert/key combo.

Reply:
Thanks, I know but I thought it is a good idea to put the cert to the secret as well, maybe for a easier debugging.